src/passes/SpillPointers.cpp

```diff
@@ -76,7 +77,12 @@ struct SpillPointers
   Type pointerType;
 
   void spillPointers() {
-    pointerType = getModule()->memories[0]->indexType;
+     // TODO: Move these two lines somewhere earlier since it only needs to be done once (but where?)
```

Comment:
A function-parallel pass like this doesn't have a "global" phase where you can look at imports once and then use that info. We could make the pass not parallel, but the number of imports is so small, and this pass is experimental, that it's probably fine to just leave it.